identity/web.py

```diff
@@ -233,6 +235,12 @@ def _get_token_for_user(self, scopes, force_refresh=None):
                 return result
         return {"error": "interaction_required", "error_description": "Cache missed"}
 
+    @functools.lru_cache(maxsize=1)
+    def _get_oidc_config(self):
+        # Emperically, Microsoft Entra ID's /v2.0 endpoint will pop up account picker
+        # but its default (i.e. v1.0) endpoint will sign out the (only?) account
+        return requests.get(f"{self._authority}/.well-known/openid-configuration").json()
```

Comment:
Are we defaulting to always using the v1 endpoint? The end session endpoint seems to vary and since the developer wouldnt typically specify the version in the authority, that would mean different end session urls..for example consider the following two documents

[https://login.microsoftonline.com/common/.well-known/openid-configuration](https://login.microsoftonline.com/common/.well-known/openid-configuration) and [https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration](https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration)

The end session endpoinjt for the two are different. One endpoint would be something like https://login.microsoftonline.com/oauth2/v2.0/logout and the other https://login.microsoftonline.com/oauth2/logout . Does that matter or do both endpoints refer to the same thing?

Reply:
Would CIAM apps require tenant specific endpoints, for example, https://login.microsoftonline.com/contoso.onmicrosoft.com/v2.0/.well-known/openid-configuration where contoso is the subdomain 

If you follow the format above, you'll end up with something like https://contoso.ciamlogin.com/.well-known/openid-configuration which wouldnt have the resource we are looking for

Reply:
> The end session endpoint seems to vary ... Does that matter or do both endpoints refer to the same thing?

They are technically not the same thing but, as mentioned in that transparent inline comment, "empirically" the two behave largely the same.

Also, we are trying to move away from hardcoding "/v2.0" in the entire stack, especially for CIAM.

> Would CIAM apps require tenant specific endpoints

Yes, and the understanding is that CIAM authority will all have their tenant present as path. So, it shall be fine. But again, please help test this (using the new Django sample).